src/parser/jobs/ast/modules/Horoscope.tsx

```diff
@@ -50,23 +61,48 @@ export default class Horoscope extends Module {
 		this.activations++
 	}
 
+	private onHeliosCast(event: CastEvent) {
+		if (this.cooldowns.getCooldownRemaining(this.data.actions.HOROSCOPE.id) > 0) {
+			return
+		}
+		this.nonHoroscopeHeals++
+	}
+
 	onComplete() {
-		/*
-			SUGGESTION: Didn't activate
-		*/
-		const missedActivations = this.uses - this.activations
-		this.suggestions.add(new TieredSuggestion({
-			icon: this.data.actions.HOROSCOPE_ACTIVATION.icon,
-			content: <Trans id="ast.horoscope.suggestion.expired.content">
-				<ActionLink {...this.data.actions.HOROSCOPE} /> does not activate by itself, so don't forget to use it again or it will expire for no potency.
+
+		if (this.parser.patch.before('5.3')){
+			/*
+				SUGGESTION: Didn't activate
+			*/
+			const missedActivations = this.uses - this.activations
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.HOROSCOPE_ACTIVATION.icon,
+				content: <Trans id="ast.horoscope.suggestion.expired.content">
+					<ActionLink {...this.data.actions.HOROSCOPE} /> does not activate by itself, so don't forget to use it again or it will expire for no potency.
 			</Trans>,
-			why: <Trans id="ast.horoscope.suggestion.expired.why">
-				<Plural value={missedActivations} one="# expiration" other="# expirations" />  of Horoscope without reading fortunes again.
+				why: <Trans id="ast.horoscope.suggestion.expired.why">
+					<Plural value={missedActivations} one="# expiration" other="# expirations" />  of Horoscope without reading fortunes again.
 			</Trans>,
-			tiers: SEVERETIES.ACTIVATES_MISSED,
-			value: missedActivations,
-		}))
+				tiers: SEVERITIES.ACTIVATES_MISSED,
+				value: missedActivations,
+			}))
 
+		} else {
+			/*
+				SUGGESTION: AOE heal without horoscope
+			*/
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.HOROSCOPE.icon,
+				content: <Trans id="ast.horoscope.suggestion.usage.content">
+					Plan to have <ActionLink {...this.data.actions.HOROSCOPE} /> up before you need to cast <ActionLink {...this.data.actions.HELIOS} /> or <ActionLink {...this.data.actions.ASPECTED_HELIOS} /> .
+				<StatusLink {...this.data.statuses.HOROSCOPE_HELIOS} /> may help to cover more damage later without needing to cast more AOE heals.
+			</Trans>,
+				tiers: SEVERITIES.WASTED_AOE_HEAL_TIERS,
+				value: this.nonHoroscopeHeals,
+				why: <Trans id="ast.synastry.suggestion.why">
+					<Plural value={this.nonHoroscopeHeals} one="# single-target GCD heal was cast" other="# single-target GCD heals were cast" /> without synastry even though it was available.
+			</Trans>,
+			}))
+		}
```

Comment:
Should the i18n keys here match up? This doesn't appear to be Synastry.

Reply:
sloppyyyyy